src/home/components/components/pokemon-detail/styles/pokemon-detail-styles.ts

```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components'
+
+// This component should stay into same shared folder
+export const CenterDiv = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+`
+export const PokemonDetailCard = styled.div`
```

Comment:
[Prettier](https://prettier.io) would really help here to have code formatted more consistent.